ganttproject/src/biz/ganttproject/impex/google/GoogleAuth.java

```diff
@@ -0,0 +1,75 @@
+package biz.ganttproject.impex.google;
+
+import com.google.api.client.auth.oauth2.Credential;
+import com.google.api.client.extensions.java6.auth.oauth2.AuthorizationCodeInstalledApp;
+import com.google.api.client.extensions.jetty.auth.oauth2.LocalServerReceiver;
+import com.google.api.client.googleapis.auth.oauth2.GoogleAuthorizationCodeFlow;
+import com.google.api.client.googleapis.javanet.GoogleNetHttpTransport;
+import com.google.api.client.http.HttpTransport;
+import com.google.api.client.json.jackson2.JacksonFactory;
+import com.google.api.client.json.JsonFactory;
+import com.google.api.client.util.DateTime;
+
+import com.google.api.services.calendar.Calendar;
+import com.google.api.services.calendar.CalendarScopes;
+import com.google.api.services.calendar.model.*;
+import com.google.common.collect.ImmutableList;
+
+import java.io.IOException;
+import java.security.GeneralSecurityException;
+import java.util.List;
+
+public class GoogleAuth {
+
+  private static final String CLIENT_ID = "299608862260-vnb8d18h7cismqil9fbhf800jlnd3u6d.apps.googleusercontent.com";
+
+  private static final String CLIENT_SECRET = "LUcqnHuf4NCqmEkljaJM0giT";
+
+  private static final String APPLICATION_NAME = "Google Export for GanttProject";
+
+  private final JsonFactory JSON_FACTORY = JacksonFactory.getDefaultInstance();
+
+  private static final List<String> SCOPES = ImmutableList.of(CalendarScopes.CALENDAR_READONLY);
+
+  public Credential authorize() throws GeneralSecurityException, IOException {
+    HttpTransport HTTP_TRANSPORT = GoogleNetHttpTransport.newTrustedTransport();
+    GoogleAuthorizationCodeFlow flow =
+        new GoogleAuthorizationCodeFlow.Builder(
+            HTTP_TRANSPORT, JSON_FACTORY, CLIENT_ID, CLIENT_SECRET, SCOPES)
+            .build();
+    return new AuthorizationCodeInstalledApp(
+        flow, new LocalServerReceiver()).authorize("ganttuser");
+  }
+
+  public Calendar getCalendarService(Credential credential) throws GeneralSecurityException, IOException {
+    HttpTransport HTTP_TRANSPORT = GoogleNetHttpTransport.newTrustedTransport();
+    return new Calendar.Builder(
+        HTTP_TRANSPORT, JSON_FACTORY, credential)
```

Comment:
pls move it to the previous line, otherwise it is on the same indent with subsequent method chaining which doesn't look good